### DISCUSSION TITLE
typo & safari to chrome compatability

### DISCUSSION THREAD

**Discussion by imogenmat** [Reading-Psych/jast#1] [Possible errors in the document] [answered]
There were a few typos in the file:
- Downloading & Installing r: ‘an’ change to ‘and’
- Packages: change ‘less’ to ‘lets’

Github also wouldn’t let me report an error when opening in safari (on MacBook - so I had to repaste into safari -not sure if this would be a problem for all MacBook users?)

**Comment by anthonyhaffey** [comment, marked as answer]
Thanks @imogenmat - I've just fixed the typos and pushed the changes.
I wasn't able to replicate the issue of the error button not working on safari on my Mac. Can you please double check whether you were logged into github on safari? If so, and it's still not working, can you please report which version of safari it was?

**Comment by imogenmat** [comment]
Hi Anthony,

Yes silly me - I hadn't logged in on safari, so all resolved 😅
Best wishes,

Imogen

________________________________

**Reply by anthonyhaffey** [reply]
No worries! You won't be the last person to have this issue, so it's helpful to leave this up for the next person. Thanks!
